src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -95,7 +95,8 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                     items.AddRange(GetListHeaderItems(span));
                 }
 
-                if (token.Parent.Parent is DocumentationCommentTriviaSyntax)
+                if (token.Parent.Parent is DocumentationCommentTriviaSyntax ||
+                    (token.Parent.Parent.IsKind(SyntaxKind.XmlEmptyElement) && token.Parent.Parent.Parent is DocumentationCommentTriviaSyntax))
```

Comment:
Just happened to notice that line 84 uses a single `&`.  Nothing to do with your change, but maybe if others have PR feedback that brings you back here you could fix that while you're here :)